versions/3.1.0.md

```diff
@@ -2290,7 +2288,7 @@ $ref: definitions.yaml#/Pet
 
 #### <a name="schemaObject"></a>Schema Object
 
-The Schema Object allows the definition of input and output data types. These types can be objects, but also primitives and arrays. This object is a superset of the [JSON Schema Draft 2019-09](http://json-schema.org/) specification.
+The Schema Object allows the definition of input and output data types. These types can be objects, but also primitives and arrays. This object is a superset of the [JSON Schema Specification Draft 2019-09](http://json-schema.org/) specification.
```

Comment:
Can we lose the final "specification"?

Reply:
I think it's correct the way it is. We don't shove Specification into the name of our project we just accept that its a project which happens to have a specification. :)

Reply:
Do you mean this is ok

> This object is a superset of the JSON Schema **Specification** Draft 2019-09 **specification**.

Or that the link should not be changed?

Reply:
@MikeRalphson @philsturgeon I changed it to make it consistent with all of the other references in the OAS spec.  I don't care whether the word is there or not, but it should either always be there or never be there.

Reply:
I misread the diff @MikeRalphson, what you said is correct. 👇 


```suggestion
The Schema Object allows the definition of input and output data types. These types can be objects, but also primitives and arrays. This object is a superset of the [JSON Schema Specification Draft 2019-09](http://json-schema.org/).
```